include/z64actor.h

```diff
@@ -310,7 +310,7 @@ typedef enum {
 
 #define ACTORCTX_FLAG_0 (1 << 0)
 #define ACTORCTX_FLAG_1 (1 << 1)
-#define ACTORCTX_FLAG_2 (1 << 2)
+#define ACTORCTX_FLAG_PICTOGRAPH_ON (1 << 2)
```

Comment:
Does this refer to having the pictograph box on? or something else?

Reply:
This is set to on when both looking through the lens in First Person to take a photo, and when looking at a photo. So `ACTORCTX_FLAG_PICTO_BOX_IN_USE`? As in using the item perhaps?

Reply:
`ACTORCTX_FLAG_PICTO_BOX_ON` works imo